src/Passes/Source/Generators/LlvmPassesConfig.hpp

```diff
@@ -0,0 +1,59 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Commandline/ConfigurationManager.hpp"
+
+namespace microsoft
+{
+namespace quantum
+{
+
+    class LlvmPassesConfiguration
```

Comment:
Consider naming identically the class and the files.

Reply:
Fixed.